.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,11 @@
+on:
+  pull_request:
+
+jobs:
+  pre-commit:
+    name: "pre-commit"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: SonarSource/gh-action_pre-commit@feature/svermeille/BUILD-3093-Provide-a-reusable-way-at-CI-level-to-enforce-pre-commits-in-the-organization #TODO: ref tag 0.0.1 after merge
```

Comment:
A trick seems to be:
- create a v1 branch equal to `feature/svermeille/BUILD-3093-Provide-a-reusable-way-at-CI-level-to-enforce-pre-commits-in-the-organization`
- use v1 ref here
- after merge, create a 1.0.0 tag
- after merge, update the v1 ref to the new tag

Reply:
I don't plan to use any v* branch for this project as this leads to more problems and maintenance efforts than it solves.
I will change this to a tagged release when this gets merged and let renovate keep it up to date :+1: